source/MaterialXRender/Mesh.cpp

```diff
@@ -205,4 +205,34 @@ void Mesh::splitByUdims()
     }
 }
 
+void MeshStream::transform(const Matrix44 &matrix)
```

Comment:
Transforms a MeshStream by a matrix. Note this is done for each element in the stream.